modules/thermal_hydraulics/src/components/HeatStructurePlate.C

```diff
@@ -38,7 +38,8 @@ HeatStructurePlate::HeatStructurePlate(const InputParameters & params)
   for (unsigned int i = 0; i < _names.size(); i++)
     _name_index[_names[i]] = i;
 
-  _material_names = getParam<std::vector<std::string>>("materials");
+  _material_names = isParamValid("materials") ? getParam<std::vector<std::string>>("materials")
+                                              : std::vector<std::string>{};
```

Comment:
use default in parameter

Reply:
same reason as HeatStructureCylindrical.C, can not use default here

Reply:
see other thread